examples/minimal-solidstart/tailwind.config.ts

```diff
@@ -1,5 +1,5 @@
 import { withUt } from "uploadthing/tw";
 
 export default withUt({
-  content: ["src/**/*.tsx"],
+  content: ["src/**/*.tsx", "../../node_modules/@uploadthing/solid/**/*.tsx"],
```

Comment:
bun doesn't put node_modules in each package so the `./node_modules/...` content path the plugin sets doesn't work here.

we should document something like

```md
While `withUT` works out of the box for many cases, you might have to add the path to the uploadthing package yourself ...
```

Reply:
alternatively we could just add some additional paths in the plugin, but that could come at the cost of some performance if we're including more paths than necessary (although i'd presume that would be marginally).

I'm thinking something like

```ts
// current
const contentPaths = [
  "./node_modules/@uploadthing/react/dist/**",
  "./node_modules/@uploadthing/solid/dist/**",
];

// new
const packages = ["react", "solid"];
const paths = packages.flatMap((pkg) => [
  `./node_modules/@uploadthing/${pkg}/dist/**`,
  `../node_modules/@uploadthing/${pkg}/dist/**`,
  `../../node_modules/@uploadthing/${pkg}/dist/**`,
]);
```

although some monorepos could be deeper, I think 2 levels are the most common and should cover 99% of cases

Reply:
I think that the cost of extra paths would be exceedingly small, so not too concerned with that route. 

`import.meta.resolve` also seems fine, worst case it doesn't work, and we fall back on the same error?
> While `withUT` works out of the box for many cases, you might have to add the path to the uploadthing package yourself ...